conda-store-server/setup.cfg

```diff
@@ -56,11 +56,7 @@ console_scripts =
     conda-store-worker=conda_store_server.worker.__main__:main
 
 [options.extras_require]
-dev = pytest; pytest-mock; pytest-playwright; black==22.3.0; flake8; sphinx; myst-parser; sphinx-panels; sphinx-copybutton; pydata-sphinx-theme; build; twine
-
-[options.packages.find]
-exclude =
-    tests
+dev = pytest; pytest-mock; pytest-sqlalchemy-mock>=0.1.4; pytest-playwright; black==22.3.0; flake8; sphinx; myst-parser; sphinx-panels; sphinx-copybutton; pydata-sphinx-theme; build; twine
```

Comment:
adding `pytest-sqlalchemy-mock` to frame/explore a way for this `conda-store-server` package to have more useful unit tests leveraging model mocks. this mostly works but the through tables are not resolving which limits its usefulness. so might remove. leaving here to see if i can solve it in the next couple days